src/aspire/operators/filters.py

```diff
@@ -33,6 +33,30 @@ def wavelength_to_voltage(wavelength):
     ) / (2 * 0.978466)
 
 
+def evaluate_grid_src(src, L, power=1):
+    """
+    Given an ImageSource object, compute the source's unique filters
+    at the filter_indices specified in its metadata.
+    :return: an `L x L x len(src.filter_indices)` array containing the evaluated
+    filters at each gridpoint
+    """
+    grid2d = grid_2d(L, dtype=src.dtype)
+    omega = np.pi * np.vstack((grid2d["x"].flatten(), grid2d["y"].flatten()))
+    h = np.empty((omega.shape[-1], len(src.filter_indices)), dtype=src.dtype)
+    for (
```

Comment:
do you know why this line's syntax changed from the old code? The old code was also under black..

Reply:
Weird, I don't even know how the parentheses got around `i, filt`. Removing them did the trick